site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 2. The source code is available in a public git repository.
```

Comment:
```suggestion
2. The source code is available in a public GitHub repository.
```

Reply:
`git` is the correct term here. E.g. it should be a GitLab repository